src/std/net/json-rpc.ss

```diff
@@ -126,22 +106,121 @@
   (def request-string (request->string method (param-encoder params) id))
   (when log
     (log [to: server-url request: request-string]))
-
   ;; TODO: implement timeouts, with semi-asynchronous aborts the http-post thread itself.
   (def response-bytes
     (request-response-bytes
      (http-post server-url
                 auth: auth
-                headers: `(("Content-Type" . "application/json") ,@headers)
+                headers: `(("Content-Type" . "application/json-rpc")
+                           ;; ("Accept" . "application/json-rpc, application/json, application/jsonrequest")
```

Comment:
?

Reply:
Not sure what you're asking. If it's about the commented out Accept, I'm adding a suitable explanation in a comment.